Add create docker images on tag

caduvieira opened the pull request.

caduvieira committed:
Add docker lint

caduvieira committed:
Deploy docker images on tag

hvelarde commented on .travis.yml:
```diff
@@ -6,47 +6,54 @@ cache:
   - eggs
   - parts/node
 addons:
-  # XXX: use an older Firefox ESR as we are stuck with selenium = 2.53.5
```
o @idgserpro quer esse comentário ai.

caduvieira replied:
quando usei o gem travis para adicionar os envs encriptados ele removeu todos os comentários não sei por que

hvelarde commented on .travis.yml:
```diff
@@ -6,47 +6,54 @@ cache:
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
 script:
-# FIXME: we have isolation issues running tests
```
eu quero esse comentário ai.

caduvieira committed:
Revert comment removal

idgserpro commented on .travis.yml:
```diff
@@ -6,32 +6,40 @@ cache:
+# DOCKER_PASS
+  - secure: Nz00qh28EYnfhT0WM32lYMt0kWkw2JcWeb3sh4ttVn6CnM5+RiN1I++4ZXC2iHD7GIrpvfSVdIie2JNr99LYNyRlGdiJv+eOO9IStGrXH8CRLyw0wzBTOItRz/pm3AYFAF9GDALrDtF3Xb8Nt+XjcfSwpFuj1fsXn4KHJp77I5E=
   matrix:
-  # TODO: update https://github.com/plonegovbr/portalpadrao.release/blob/master/README.md
```
Favor não remover o comentário.

idgserpro commented on .travis.yml:
```diff
@@ -6,32 +6,40 @@ cache:
 before_install:
-- test $DEVELOPMENT && cp travis-development.cfg buildout.cfg || cp travis-production.cfg buildout.cfg
+- test $DEVELOPMENT && cp travis-development.cfg buildout.cfg || cp travis-production.cfg
+  buildout.cfg
```
O `buildout.cfg` tem que ficar na linda de cima.

idgserpro commented on .travis.yml:
```diff
@@ -6,32 +6,40 @@ cache:
 addons:
-  # XXX: use an older Firefox ESR as we are stuck with selenium = 2.53.5
-  #      https://github.com/SeleniumHQ/selenium/issues/2739#issuecomment-249482533
+# XXX: use an older Firefox ESR as we are stuck with selenium = 2.53.5		
```
Remover os espaços.

idgserpro commented on .travis.yml:
```diff
@@ -6,32 +6,40 @@ cache:
+# Dessa forma, passamos a testar se o egg foi realmente adicionado no buildout		
+# para que esse erro não ocorra novamente.		
+# Caso altere a versão do zc.buildout, pode ser que você necessite alterar o		
+# arquivo onde procura a string, trocando bin/instance por parts/instance.site.py		
```
Remover os espaços.

idgserpro commented on .travis.yml:
```diff
@@ -6,32 +6,40 @@ cache:
-  # XXX: use an older Firefox ESR as we are stuck with selenium = 2.53.5
-  #      https://github.com/SeleniumHQ/selenium/issues/2739#issuecomment-249482533
+# XXX: use an older Firefox ESR as we are stuck with selenium = 2.53.5		
+#      https://github.com/SeleniumHQ/selenium/issues/2739#issuecomment-249482533
```
Indentar o comentário.

idgserpro committed:
Merge branch 'master' into docker

idgserpro commented:
@caduvieira os testes estão falhando.

hvelarde commented:
@caduvieira tu vai continuar com isso aqui?

hvelarde committed:
Fail fast on errors

hvelarde committed:
Update collective.lazysizes to 4.1.2

hvelarde committed:
Update collective.cover to 1.8b2

hvelarde committed:
Update brasil.gov.barra to 3.0.5

rodfersou committed:
Update static resources

rodfersou committed:
Update static resources to version 1.1b5

hvelarde committed:
Update brasil.gov.temas to 2.0b4

hvelarde committed:
Update brasil.gov.portal to 2.1.1b1

hvelarde committed:
Use release 2.1b1

hvelarde committed:
Use brasil.gov.portal from master branch

hvelarde committed:
Use brasil.gov.temas from master branch

hvelarde committed:
Update collective.lazysizes to 4.1.4

hvelarde committed:
Upgrade collective.fingerpointing to 1.8

Add product-config option to production.cfg.

hvelarde committed:
Update brasil.gov.temas to 2.0b5

hvelarde committed:
Update brasil.gov.portal to 2.1rc1

hvelarde committed:
Use release 2.1rc1-pending

hvelarde committed:
Release 2.1rc1 as final

hvelarde committed:
Use brasil.gov.tiles from master branch

hvelarde committed:
Use brasil.gov.agenda from master branch

caduvieira commented:
Estou sem tempo para continuar. Pode fechar

hvelarde commented:
vou deixar aberto, tal vez alguém se anima continuar.

idgserpro committed:
Adiciona comando "cd portal.buildout"

idgserpro committed:
Remove comando "cd portal.buildout"

O comando já estava mais abaixo na documentação.

hvelarde committed:
Use brasil.gov.temas from master branch

hvelarde committed:
Use brasil.gov.portal from master branch

hvelarde committed:
Remove needless port-base directive from instance part

hvelarde committed:
Update brasil.gov.barra to 3.0.6

hvelarde committed:
Clean up ZEO configuration

Remove eggs directive as it breaks the usage of ZRS.

Also, remove needless zeo-conf-additional directive.

hvelarde committed:
Pin versions for ZRS

hvelarde committed:
Fix ZRS documentation

hvelarde committed:
Upgrade brasil.gov.agenda to 2.0b1

hvelarde committed:
Upgrade brasil.gov.tiles to 2.0rc1

hvelarde committed:
Upgrade brasil.gov.temas to 2.0rc1

hvelarde committed:
Update brasil.gov.portal to 2.1rc2

hvelarde committed:
Use release 2.1rc2

hvelarde committed:
Use brasil.gov.temas from master branch

hvelarde committed:
Use brasil.gov.portal from master branch

hvelarde committed:
Upgrade brasil.gov.temas to 2.0

hvelarde committed:
Update brasil.gov.portal to 2.1

hvelarde committed:
Use release 2.1

hvelarde committed:
Update brasil.gov.temas to 2.0.1

hvelarde committed:
Update brasil.gov.portal to 2.1.1

hvelarde committed:
Use release 2.1.1

idgserpro committed:
Atualiza versão do brasil.gov.temas;

idgserpro committed:
Atualiza brasil.gov.agenda para 2.0b2;

idgserpro committed:
Pina lazy-object-proxy: fix build #2558 e #2557

Corrige https://github.com/plonegovbr/brasil.gov.portal/issues/586.

idgserpro committed:
Merge pull request #131 from plonegovbr/issue-586-brasil-gov-portal

Pina lazy-object-proxy: fix build #2558 e #2557

caduvieira committed:
Corrigi docker para versão 2.1.1

caduvieira committed:
Rebase

caduvieira committed:
Corrigi latest versão do pacote

caduvieira commented:
Como posso corrigir essa questão da barra?

idgserpro commented:
O código da barra no `brasil.gov.barra` está diferente do código da barra na internet. É necessário atualizar o código da barra no `brasil.gov.barra`.

Só de rodar o buildout do `brasil.gov.barra` ele atualiza o código da barra.

Isso foi feito, por exemplo nesse commit:

https://github.com/plonegovbr/brasil.gov.barra/commit/4bdcc702290e32d896636e2df5de3f10266be305
